dist/index.js

```diff
@@ -47,8 +47,7 @@ exports.templateDefaults = {
     command: 'git checkout -b {branch | shellquote} && git commit --allow-empty -m {commit | shellquote}',
 };
 var renderer = function (templates, name) {
-    var completeTemplates = __assign(__assign({}, exports.templateDefaults), { templates: templates });
-    var render = template_1.default(completeTemplates[name], helpers);
+    var render = template_1.default(templates[name] || exports.templateDefaults[name], helpers);
```

Comment:
This is the build output in `dist/` which is currently added to the repo so the package can be used in other projects without it having to be compiled. I think that's something that we should look into however, since it adds a lot of noise.

For comparison, this is how `ky` handles this (it is a package built from TypeScript sources too):

- https://github.com/sindresorhus/ky/blob/v0.28.0/package.json#L13-L28
- https://github.com/sindresorhus/ky/blob/v0.28.0/.gitignore#L1